CHANGELOG.md

```diff
@@ -4,7 +4,7 @@
 ### Features
 - N/A
 ### Bug Fixes
-- Fixed an issue where the connector would try to download a server certificate before the WAN was initialized. 
+- Fixed an issue where the connector would try to download a server certificate before the WAN was initialized. This will resolve mqtt related https errors on cellular connections. 
```

Comment:
```suggestion
- Fixed an issue where the connector would try to download a server certificate before the WAN was initialized. This will resolve MQTT-related HTTPS errors on cellular connections. 
```

Reply:
Made this change. Also modified the jvmrun and package.html